src/System.Private.CoreLib/shared/System/Collections/Generic/List.cs

```diff
@@ -27,7 +27,10 @@ public class List<T> : IList<T>, IList, IReadOnlyList<T>
         private int _size; // Do not rename (binary serialization)
         private int _version; // Do not rename (binary serialization)
 
-        private static readonly T[] s_emptyArray = new T[0];
+        // private static readonly T[] s_emptyArray = new T[0];
+
+        // DO NOT MERGE: for testing only 
+        private static readonly T[] s_emptyArray = System.GC.AllocateUninitializedArray<T>(0); // why not
```

Comment:
> // why not

What's the benefit over `new T[0]` in this case, since there's nothing to avoid initializing?

Reply:
This is for testing. Ignore this. 
`System.GC.AllocateUninitializedArray<T>(0)` is permitted, but not the wisest thing to do in real code.